examples/client_utils/case_list.py

```diff
@@ -0,0 +1,18 @@
+import flow360.v1 as fl
+from flow360.log import log
+
+for case in fl.MyCases(limit=10000):
+    log.info(
+        "\n"
+        + "Case ID: "
+        + str(case.id)
+        + "\n"
+        + "Case name: "
+        + str(case.name)
+        + "\n"
+        + "Status: "
+        + str(case.status.value)
+        + "\n"
+        + "Solver version: "
+        + str(case.solver_version)
+    )
```

Comment:
use: https://github.com/flexcompute/Flow360/blob/1f257d70886109f20a895384448df3e3d063dbd2/flow360/component/resource_base.py#L344C9-L344C26
you can append solver version in example only